bahmnicore-omod/src/main/resources/liquibase.xml

```diff
@@ -4807,6 +4807,111 @@
+        <sql>
+            DELETE FROM concept_set WHERE concept_set.concept_set=(SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true)
+            AND concept_set.concept_id=(SELECT concept_id FROM concept_name WHERE name = 'Capsule(s)' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true);
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-202303201220" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <sqlCheck expectedResult="0">
+                    SELECT count(concept_set_id) FROM concept_set WHERE concept_set = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true)
+                    AND concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Tablet(s)' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true);
+                </sqlCheck>
+            </not>
+            <not>
+                <sqlCheck expectedResult="1">
+                    SELECT count(concept_set_id) FROM concept_set WHERE concept_set = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true)
+                    AND concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Tablet(s)' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true);
+                </sqlCheck>
+            </not>
+        </preConditions>
+        <comment>Delete Tablet(s) from Dosing Unit concept set</comment>
+        <sql>
+            DELETE FROM concept_set WHERE concept_set.concept_set=(SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true)
+            AND concept_set.concept_id=(SELECT concept_id FROM concept_name WHERE name = 'Tablet(s)' AND concept_name_type = 'FULLY_SPECIFIED' AND locale_preferred=true);
+        </sql>
+    </changeSet>
+    <changeSet id="bahmni-core-202303200330" author="Parvathy, Rahul">
+        <preConditions onFail="MARK_RAN">
+            <sqlCheck expectedResult="0">
+                SELECT count(concept_set_id) FROM concept_set WHERE concept_set = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED')
+                AND concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Tablet' AND concept_name_type = 'FULLY_SPECIFIED');
+            </sqlCheck>
+        </preConditions>
+        <comment>Add Tablet to Dosing Unit concept set</comment>
+        <sql>
+            SET @set_concept_id = (SELECT concept_id FROM concept_name WHERE name = 'Dosing Units' AND concept_name_type = 'FULLY_SPECIFIED');
+
+            SET @tablet_concept_id = (SELECT concept_id FROM concept_name where name = 'Tablet' AND concept_name_type = 'FULLY_SPECIFIED');
```

Comment:
tablet(s) should be renamed as Tablet right ? is it possible to handle as part of one migration where you are deleting Tablet(s) ?